ProcessList.c

```diff
@@ -364,4 +537,14 @@ void ProcessList_scan(ProcessList* this, bool pauseProcessUpdate) {
          p->updated = false;
       }
    }
+
+   // clean up hashtable in order to not have
+   // processes from privious build in it
+   //
+   // sorting algorithm relayes on the fact that
+   // len(this->displayTreeSet) == len(this->processes)
+   Hashtable_delete(this->displayTreeSet);
+   this->displayTreeSet = Hashtable_new(4096, false);
+
+   ProcessList_buildTree(this);
```

Comment:
I guess this should not be called in sorted-mode.

Reply:
I probably haven't get the question correctly but it should as here me clean the previous hashtable in order to build new one and don't have any artifacts from old one.

Imagine the first build have 100 processes but the second 50. If we wouldn't clean this up we would have a incorrect hashtable

Reply:
In *sorted*-mode (opposed to *tree*-mode (toggle-able with F5)) I think we do not need to build the tree (cause it is not used).